CLOUDP-172619: Update AKO to 1.7.2

helderjs opened the pull request:
## Proposed changes

The latest release of Atlas operator has some changes in CRDs and to have proper exported values from generate/apply commands we need to bump the AKO version to 1.7.2

_Jira ticket:_ CLOUDP-172619

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

gssbzn reviewed (dismissed):
LGTM but can you please take a look at https://jira.mongodb.org/browse/CLOUDP-172516 the release broke some tests (now fixed on this PR) just in case there's something we should do to prevent those errors

helderjs commented:
> LGTM but can you please take a look at https://jira.mongodb.org/browse/CLOUDP-172516 the release broke some tests (now fixed on this PR) just in case there's something we should do to prevent those errors

@gssbzn 
When I updated the AKO version the test didn't pass. This happened because we only map the latest 3 major versions of the operator and any minor version. To avoid this happening again I added this function https://github.com/mongodb/mongodb-atlas-cli/pull/1889/files#diff-622cfc041b4c9833950c6f3cf578708c64930ac93086288b03a34c7babf18cbeR262 thas always returns the major version of the identified operator version.
